cid/helpers/quicksight/__init__.py

```diff
@@ -238,6 +238,28 @@ def discover_dashboard(self, dashboardId: str) -> Dashboard:
             templateAccountId = _definition.get('sourceAccountId')
             templateId = _definition.get('templateId')
             region = _definition.get('region', 'us-east-1')
+
+            # Checking for version override in template definition 
+                 
+            version = dashboard.deployedTemplate.version
+            
+            version_obj = _definition.get('versions')
+            
+            if not version_obj: raise CidCritical("No `versions` key find in dashboard definition resources.yml")
+            
+            version_map = version_obj.get('versionMap', dict())
+            
+            description_override = version_map.get(version)
```

Comment:
This may throw error if versionMap is not there

Reply:
No dict.get return None when the key is not found without throwing errors